pkg/volume/glusterfs/glusterfs.go

```diff
@@ -240,7 +240,12 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 	if err := os.MkdirAll(p, 0750); err != nil {
 		return fmt.Errorf("glusterfs: mkdir failed: %v", err)
 	}
-	log := path.Join(p, "glusterfs.log")
+
+	// adding log-level ERROR to remove noise
+	// and more specific log path so each pod has
+	// it's own log based on PV + Pod
+	log := path.Join(p, b.pod.Name + "-glusterfs.log")
```

Comment:
@rootfs - yeah, I was thinking of those cases, although probably rare, but could have multiple pods using the same PV and would have the log clobbered and maybe even hard to pin point specific errors - so was thinking of a way to better isolate the log but still have the user be able to easily find it.  Maybe it's over kill?  The pod name can not be longer than 253 characters, so what if I substring'd the pod if it was over X length to first 32 characters?